tests/showImages.js

```diff
@@ -106,6 +106,18 @@ module.exports = {
 			.assert.attributeEquals('.res-expando-box .res-gallery-pieces > div:not([hidden]) a', 'href', 'https://imgur.com/rXZWEIB,eutVEAv#eutVEAv')
 			.end();
 	},
+	'crosspost expando': browser => {
+		browser
+			.url('https://en.reddit.com/by_id/t3_7fsw5w/')
+			.waitForElementVisible('.expando-button')
+			.click('.expando-button')
+			.waitForElementVisible('.res-expando-box')
+			.assert.cssClassPresent('.crosspost-preview', 'res-crosspost-preview')
+			.assert.containsText('.crosspost-preview .title', 'Original post with expandoable image for crossposting')
+			.assert.attributeEquals('.crosspost-preview a.author', 'href', '/user/andytubatest/')
```

Comment:
The original author is /u/andytuba, so this should be /user/andytuba/, I believe.

You may also need to use `attributeContains`, since I think it does the `element.href` URL expansion (i.e. it's not just `element.getAttribute('href')` like the name suggests...)